ugc/storage.cpp

```diff
@@ -72,6 +72,61 @@ string SerializeUGCIndex(vector<Storage::UGCIndex> const & indexes)
   unique_ptr<char, JSONFreeDeleter> buffer(json_dumps(array.get(), JSON_COMPACT | JSON_ENSURE_ASCII));
   return string(buffer.get());
 }
+
+template <typename UGCUpdate>
+Storage::SettingResult SetGenericUGCUpdate(
+    vector<Storage::UGCIndex> & indexes, size_t & numberOfDeleted, FeatureID const & id,
+    UGCUpdate const & ugc,
+    unique_ptr<FeatureType> featureType,
```

Comment:
Why not to pass featureType by const ref here?

Reply:
Yes, I agree. Good idea.